DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>Does this need its own cell? Could just print from the earlier cell once you do the denomination fix</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>